express-post-json/server.ts

```diff
@@ -0,0 +1,41 @@
+import express from 'express';
+
+type NextId = number;
```

Comment:
This is fine, but overkill. The reason is that on line 12, which is the only place you use this type, TypeScript can already figure out that `nextId` is a number since you are assigning it the value 1.

Reply:
Updated